velox/functions/lib/string/StringImpl.h

```diff
@@ -309,7 +309,7 @@ FOLLY_ALWAYS_INLINE bool md5_radix(
 
 namespace {
 FOLLY_ALWAYS_INLINE int64_t asciiWhitespaces() {
-  std::vector<int32_t> codes = {9, 10, 11, 12, 13, 28, 29, 30, 31, 32};
+  std::vector<int8_t> codes = {9, 10, 11, 12, 13, 28, 29, 30, 31, 32};
```

Comment:
Let's just use array on stack: `int8_t codes[] = {...}` since there is only 9 bytes